paperqa/agents/chains.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+import logging
+import re
+
+from openai import AsyncOpenAI
+
+from paperqa import OpenAILLMModel
+
+from .tools import get_year
+
+logger = logging.getLogger(__name__)
+
+
+async def search_chain(
```

Comment:
Maybe this is my opinion, but can we not call them "chains"? It's just one LLM prompt, right?

I know that's a LangChain thing, but tbh it's just an API call with some pre/post processing

Reply:
yea, legacy func for sure